### ISSUE TITLE
Need to add one comment after every 100 comments delete

### ISSUE THREAD

**Issue by Indratejreddy** [kennethkn/instagram-activities-wipe#2]
I found that instagram algorithm is stopping us by not showing us next comments but we can hack the algo by adding one comment after every delete

**Comment by kennethkn** [comment]
Hi there! Thanks for raising the issue. Could you elaborate on the details of your circumstances where the hack worked? I want to make sure we are talking about the same issue. 

Were you getting the "something went wrong" prompt, where it does not load the next comments properly after you hit the rate limit, and even after a reload the same thing happens until a few hours pass? 

If so, are you saying that adding a comment bypasses the limit such that you don't have to wait a few hours before retrying?

To be honest, I'm out of comments to test with because the script has removed them all for me. So your input will help a bunch if you could described how your solution works in details. So that I can have enough information to implement it in the script.

Thanks for letting me know!

**Comment by googledino** [comment]
I just tried it myself and I can confirm; instagram was telling me I had no comments posted yet, I went to a random post and made a comment, and all of a sudden the older comments loaded again

**Comment by Indratejreddy** [comment]
> I just tried it myself and I can confirm that Instagram was telling me I had no comments posted yet, I went to a random post and made a comment, and all of a sudden the older comments loaded again

yes but if you add one comment manually from your phone or web and refresh it will show. so we need to modify the script in such a way that it automatically add 1 comment after deleting 100 comments
